blueprints/incredible-squaring-symbiotic/src/lib.rs

```diff
@@ -17,8 +18,9 @@ load_abi!(
     "contracts/out/IncredibleSquaringTaskManager.sol/IncredibleSquaringTaskManager.json"
 );
 
-pub fn noop(_: U256) {
+pub async fn noop(_: U256) -> Result<(), gadget_sdk::Error> {
```

Comment:
Would the default post processor be a noop? Did we not integrate that yet?

Reply:
Yes, not including the post processor will default to no-op. I can remove this here.